aztec/src/main/kotlin/org/wordpress/aztec/Aztec.kt

```diff
@@ -104,20 +133,20 @@ open class Aztec private constructor(val visualEditor: AztecText, val sourceEdit
         plugins.add(plugin)
 
         if (plugin is IToolbarButton) {
-            toolbar.addButton(plugin)
+            toolbar?.addButton(plugin)
         }
 
         return this
     }
 
     fun initHistory() {
-        sourceEditor.history = visualEditor.history
+        sourceEditor?.history = visualEditor.history
     }
 
     private fun initToolbar() {
-        toolbar.setEditor(visualEditor, sourceEditor)
-        toolbar.setToolbarListener(toolbarClickListener)
-        visualEditor.setToolbar(toolbar)
+        toolbar?.setEditor(visualEditor, sourceEditor!!)
```

Comment:
The `setEditor()` function will need to be modified so that it can handle the case without the source editor.